src/ape/contracts/base.py

```diff
@@ -333,6 +334,8 @@ def call(self) -> ContractCallHandler:
     def __call__(self, *args, **kwargs) -> ReceiptAPI:
         function_arguments = self._convert_tuple(args)
         contract_transaction = self._as_transaction(*function_arguments)
+        if "sender" not in kwargs and ape.accounts.default_sender is not None:
```

Comment:
Because this class subclasses `ManagerAccessMixin`, we don't need to import `ape` (which causes a cycle)
```suggestion
        if "sender" not in kwargs and self.account_manager.default_sender is not None:
```